tests/network/probes.go

```diff
@@ -252,15 +250,6 @@ func createGuestAgentPingProbe(period, initialSeconds int32) *v1.Probe {
 	return createProbeSpecification(period, initialSeconds, 1, handler)
 }
 
-func createHTTPProbe(period, initialSeconds int32, port int) *v1.Probe {
```

Comment:
This function should be removed in the following commit.
Otherwise, this code from this commit will not compile.